### PR TITLE
Add direct F# support to .net sdk 1.1.0

### PR DESCRIPTION
**Pull request by KevinRansom**
Build verification.

Enable direct support for F# in dotnet sdk 1.1

### CODE REVIEW

**Comment by KevinRansom** [comment]
@nguerrera @livarcocc 
Can you take a look guys please.

File: src/Tasks/Microsoft.NET.Build.Tasks/build/Microsoft.NET.Sdk.FSharp.targets [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+  <PropertyGroup  Condition=" '$(UseBundledFSharpTargets)' == 'true' ">
+    <DefineConstants>$(DefineConstants);$(ImplicitFrameworkDefine)</DefineConstants>
+  </PropertyGroup>
+
+  <!-- ***************************************************************************************************************
+       Loads the cross-targeting targets if we are doing a cross-targeting build
+       Import design time targets for Roslyn Project System. These are only available if Visual Studio is installed.
+       Import design time targets before the common crosstargeting targets, which import targets from Nuget.
+       *************************************************************************************************************** -->
+  <PropertyGroup Condition=" '$(IsCrossTargetingBuild)' == 'true' " >
```
Comment (srivatsn) [inline src/Tasks/Microsoft.NET.Build.Tasks/build/Microsoft.NET.Sdk.FSharp.targets]: This shouldn't be conditioned on IsCrossTargetingBuild

**Comment by KevinRansom** [comment]
@dotnet-bot test Windows_NT_FullFramework Release

**Comment by KevinRansom** [comment]
@dotnet-bot test Ubuntu14.04 Debug please

**Comment by KevinRansom** [comment]
@dotnet-bot test this please

**Comment by KevinRansom** [comment]
@dotnet-bot test Windows_NT_FullFramework Release please

**Comment by nguerrera** [comment]
With the feedback applied, is this file now identical to the copy in release/2.0.0? If not, please submit another PR to 2.0 to sync them up.

**Review by nguerrera — APPROVED** [review]

**Comment by nguerrera** [comment]
Fill out the ask mode template and tag Matt.

**Comment by KevinRansom** [comment]
**Tagging @MattGertz for approval pending CI tests.**

**Customer scenario**:    VS ships with the V1.1 sdk and dotnet CLI without this change netstandard libraries and apps will not be editable in VS.

**Workarounds, if any**:     None.

**Risk:**:     Low

**Performance impact** : None only enabled for .fsproj

**Is this a regression from a previous update?:**, dotnet core support hasn't been provided in the past.

**Root cause analysis** : Not yet implemented

**How was this bug found?:** : Not yet implemented

**Comment by nguerrera** [comment]
> Workarounds, if any: None.

Installing 2.0 CLI would be a workaround, no?

**Comment by KevinRansom** [comment]
@nguerrera agreed

**Review by livarcocc — APPROVED** [review]

**Pull request merged**